app/terminal/terminal.tsx

```diff
@@ -10,18 +10,15 @@ interface TerminalState {
   wrap: boolean;
 }
 
-const wrapLocalStorageKey = "terminal-wrap";
-const wrapLocalStorageValue = "wrap";
+const WRAP_LOCAL_STORAGE_KEY = "terminal-wrap";
+const WRAP_LOCAL_STORAGE_VALUE = "wrap";
+const ANSI_STYLES_REGEX = /\x1b\[[\d;]+?m/g;
 
 export default class TerminalComponent extends React.Component<TerminalProps, TerminalState> {
-  state = { wrap: false };
+  state = { wrap: localStorage.getItem(WRAP_LOCAL_STORAGE_KEY) === WRAP_LOCAL_STORAGE_VALUE };
 
   terminalRef = React.createRef<HTMLDivElement>();
 
-  componentDidMount() {
-    this.setState({ wrap: localStorage.getItem(wrapLocalStorageKey) === wrapLocalStorageValue });
```

Comment:
I initially had this in the constructor, but I moved this here because `wrapText` calls getBoundingClientRect to get the width of the terminal - and `this.terminalRef.current` wasn't available at construction time.

Reply:
Gotcha, reverted

Reply:
If you wanted to avoid the double render on the non-wrap case, you could check that wrap is enabled before calling setState. Not sure how bad the double render is in practice.